src/main/java/eu/seal/idp/service/impl/SAMLDatasetDetailsServiceImpl.java

```diff
@@ -27,25 +34,65 @@ public DataSet loadDatasetBySAML(String dsId, SAMLCredential credential)
 		
 		//dataSet.setLoa(user.getLoa()); To be set 
         //dataSet.setIssued(id);
+		String id = UUID.randomUUID().toString();	
 		SimpleDateFormat formatter = new SimpleDateFormat("EEE, d MMM YYYY HH:mm:ss z", Locale.US);
         formatter.setTimeZone(TimeZone.getTimeZone("GMT"));
         Date date = new Date();
         String nowDate = formatter.format(date);
 		DataSet dataSet = new DataSet();
+		dataSet.setId(id);
         dataSet.setIssuerId("This is the user ID.");
         dataSet.setIssued(nowDate);
-        dataSet.setType("eIDAS");
+        dataSet.setType("eduGAIN");
+
 		
 		List<Attribute> attributesList = credential.getAttributes();
 		
 		for (Attribute att: attributesList) {
 			AttributeType attributeType = new AttributeType();
+			LOG.info("*****_Name" + att.getName());
 			attributeType.setName(att.getName());
 			attributeType.setFriendlyName(att.getFriendlyName());
+			LOG.info("*****_FriendlyName" + att.getName());
 			dataSet.addAttributesItem(attributeType);
 		}
 		
 		LOG.info(dataSet.toString());
 		return dataSet;
 	}
+	
+	public AttributeSet loadAttributeSetBySAML(String dsId, String inResponseTo, SAMLCredential credential)
+			throws UsernameNotFoundException {
+		
+		AttributeSet attributeSet = new AttributeSet();
+		attributeSet.setId(dsId);
+		List <AttributeType> attributes = new ArrayList();
+		AttributeType[] attributeTypeArray = new AttributeType[attributes.size()];
+		List<Attribute> attributesList = credential.getAttributes();
+		
+		for (Attribute att: attributesList) {
+			AttributeType attributeType = new AttributeType();
+			attributeType.setName(att.getName());
+			attributeType.setFriendlyName(att.getFriendlyName());
+			attributes.add(attributeType);
+		}
+	    AttributeSetStatus atrSetStatus = new AttributeSetStatus();
+	    atrSetStatus.setCode(AttributeSetStatus.CodeEnum.OK);
+	    
+		AttributeSet attrSet = new AttributeSet();
+		attrSet.setId(UUID.randomUUID().toString());
+		attrSet.setType(TypeEnum.RESPONSE);
+		attrSet.setIssuer("edugainASms_001");
```

Comment:
Be aware! Use an env variable instead?

Reply:
I think this comment is on an older commit! Take a look at the message above where I put the lates status

Reply:
OK